docs/ref/databases.txt

```diff
@@ -113,6 +113,27 @@ PostgreSQL connection settings
 
 See :setting:`HOST` for details.
 
+To connect using the service name of your PostgreSQL database, your ``settings.py``
+file should look something like this::
+
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'OPTIONS': {'service': 'my_service'},
+        }
+    }
+
+Here is a sample ``.pg_service.conf`` file::
+
+    [my_service]
+    host=localhost
+    user=postgres
+    dbname=postgres
+    password=postgres
+    port=5432
+
+.. versionadded:: 3.2
```

Comment:
This should be `versionchanged` with a few words of explanation: `Support for ....`